ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -160,6 +155,17 @@ const EnableCtSwitch = (props: {
   );
 };
 
+const MembershipRequirements = [
```

Comment:
this was the only remaining piece of admin-institution-options, so I moved it in here